Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -20,30 +20,99 @@ Write-Verbose "cleanTargetBeforeCopy = $cleanTargetBeforeCopy" -Verbose
 
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
 
-$connection = Get-VssConnection -TaskContext $distributedTaskContext
+	# Default + constants #
+$defaultWirmPort = '5985'
+$defaultSkipCACheckOption = '-SkipCACheck'
+$defaulthttpProtocallOption = '-UseHttp'
+$resourceFQDNKeyName = 'Microsoft-Vslabs-MG-Resource-FQDN'
+$resourceWinRMHttpPortKeyName = 'WinRM_HttpPort'
+$resourceWinRMHttpsPortKeyName = 'WinRM_HttpsPort'
```

Comment:
Should the values for these KeyNames come from a constant defined in DevTestLabs dll or a Service dll? I would like to keep them closer to Machine group and refer them here to avoid any future compat issues.